en/02_Developer_Guides/18_Cookies_And_Sessions/02_Sessions.md

```diff
@@ -124,7 +124,7 @@ SilverStripe\Core\Injector\Injector:
 
 ### Secure session cookie
 
-A different `session_name` cookie is used when using the `https` protocol for security purposes. This uses the session_name `SECSESSID` for `https` connections instead of the default `PHPSESSID`. Doing so adds an extra layer of security to your session cookie since you no longer share `http` and `https` sessions.
+A different `session_name` cookie is used for `https` and `http` connections for security purposes. While the session name `PHPSESSID` remains the same for both `https` and `http` connections, the `https` connection automatically includes the `Secure` attribute. This ensures that the `https` session cookie is only transmitted over encrypted HTTPS connections, preventing it from being exposed during plain-text HTTP requests and enhancing overall security. This means that if you are serving your site over both `https` and `http`, things may not work as expected.
```

Comment:
```suggestion
The session cookie settings vary slightly between `HTTP` and `HTTPS` connections. `HTTPS` connections automatically include the `Secure` attribute. This ensures that secure session cookie data is only transmitted over encrypted `HTTPS` connections, preventing it from being exposed during plain-text `HTTP` requests and enhancing overall security. This means that if you are serving your site over both `HTTPS` and `HTTP`, some functionality such as authentication may not work as expected.
```
The name is no longer relevant for this documentation I think.

Not sure if HTTP and HTTPS should be backticked or not but it should be consistent either way.

Reply:
Updated